modules/db.ts

```diff
@@ -123,7 +123,42 @@ const setInitPurchasedMiddleware = {
   },
 };
 
+/** Use Web Crypto API to set ID */
+const uuidMiddleware = {
+  stack: 'dbcore' as const,
+  name: 'uuidMiddleware',
+  create: (downlevelDBCore: DBCore): DBCore => {
+    return {
+      ...downlevelDBCore,
+      table: (tableName: string): DBCoreTable => {
+        const downlevelTable = downlevelDBCore.table(tableName);
+
+        return {
+          ...downlevelTable,
+          mutate: async (req: DBCoreMutateRequest) => {
+            if (
+              tableName === 'items' ||
+              tableName === 'lists' ||
+              tableName === 'itemLists'
```

Comment:
There should be a more succinct way to do this, because we're doing it for every table. This will get annoying to maintain in the future.